_sass/partials/_grid.scss

```diff
@@ -44,5 +46,12 @@
 .split--half {
   @media screen and (min-width: $screen-sm-min) {
     @include flex(0 0 50%);
+
+    &:first-of-type {
+      padding-right: $u-space-default;
+    }
```

Comment:
would add space in cases like this (nitpick, feel free to ignore stuff like this)